addons/hide_private_properties/inspector_plugin.gd

```diff
@@ -1,10 +1,19 @@
 extends EditorInspectorPlugin
 
 func _can_handle(object: Object) -> bool:
+    if not _has_property(object, "scene_file_path"):
```

Comment:
Could you leave a comment above this line why this check is required? Just one or two sentence.

Reply:
Pushed new commit with comments added.

- added short explanation above the if on why the check is done 
- added note in `_has_property` on why using a for loop instead of the `in` keyword for checking if the property exists